### ISSUE TITLE
FRESH DROPLET After deleting site and recreating - not able to add to database

### ISSUE THREAD

**Issue by shaneholloman** [rtCamp/easyengine#455]
On a fresh droplet and EE installation
```
root@loophole:/# ee site create loophole.io --wpfc
Setting up NGINX configuration 	[Done]
Setting up webroot 		[Done]
Downloading Wordpress 		[Done]
Setting up database		[Done]
Installing plugin nginx-helper
Installing plugin w3-total-cache
Reload : nginx     [OK]
WordPress admin user : loophole
WordPress admin user password : XXXXXXXXXXXX
Configure nginx-helper:	http://loophole.io/wp-admin/options-general.php?page=nginx
Configure W3TC:		http://loophole.io/wp-admin/admin.php?page=w3tc_general
Page Cache:		Disable
Database Cache:		Memcached
Object Cache:		Memcached
Browser Cache:		Disable
Unable to add site to database
root@loophole:/#
```

**Comment by shaneholloman** [comment]
`Unable to add site to database` this only appears to be a notice which is not valid as site was in fact added to the database... so this is just probably a little house keeping

**Comment by rahul286** [comment]
This is a known issue.

**Comment by harshadyeola** [comment]
@shaneholloman @rahul286 :
This issue is resolved in EasyEngine v3.0.7. `ee site create` command is updated with cleanup actions.
If `ee site create` command fails at any step it will undo the above steps so that user will be able to create site with same domain name.
